### ISSUE TITLE
whitelist functions from base and recommended packages

### ISSUE THREAD

**Issue by luckyrandom** [jimhester/lintr#87]
such as `.onAttach`, `.onLoad`, and don't complain that they are nor all lower case, since they are required by CRAN.

**Comment by jimhester** [comment]
Yes I am aware of this bug, and it should include function names from all recommended packages.

In the meantime before it is fixed you can selectively ignore the lints from this by appending `# nolint` to the line so the lints are excluded.

**Comment by jimhester** [comment]
https://github.com/wch/r-source/blob/fbf5cdf29d923395b537a9893f46af1aa75e38f3/share/make/vars.mk#L6-L12 has the list of packages.

**Comment by nsh87** [comment]
How do you append `# nolint` to the line without causing a different linter to fail?
```
 [[2]]
./R/testpackage.R:23:39: style: Opening curly braces should never go on their own line and should always be followed by a new line.
.onLoad <- function(libname, pkgname) {  # nolint
```

**Comment by jimhester** [comment]
I doubt the crab version has exclusions implemented you need to use the github version.

```r
devtools::install_github("jimhester/lintr")
```

**Comment by nsh87** [comment]
It's "unavailable for R 3.2.0", the latest version. Any workaround or way you can update that?

**Comment by nsh87** [comment]
Hmm, never mind, it's working now.
